spec/models/metric_spec.rb

```diff
@@ -187,13 +187,13 @@
           expect(MiqQueue.count).to eq(10)
 
           expected_targets = Metric::Targets.capture_targets(nil, :exclude_storages => true)
-          expected = expected_targets.collect { |t| [t, "historical"] * 2 }.flatten # Vm, Host, Host, Vm, Host
+          expected = expected_targets.flat_map { |t| [t, "historical"] * 2 } # Vm, Host, Host, Vm, Host
```

Comment:
Same goes here (and probably the other flat_maps and flattens as well need to be reviewed in the sense of https://github.com/ManageIQ/manageiq/pull/6682/files#r53031469